api/app/signals/apps/signals/models/area.py

```diff
@@ -20,6 +21,7 @@ class Meta:
         verbose_name = 'Gebied'
         verbose_name_plural = 'Gebieden'
         unique_together = ('code', '_type')
+        ordering = ['_type', 'code']
```

Comment:
Forgot to run `makemigrations`?

There are now changes in the model that are not reflected in a migration. Please add the migration.

Reply:
will fix